src/index.js

```diff
@@ -44,6 +44,9 @@ class Popover {
     });
 
     this.textarea = textarea;
+    // TODO: Check if CSP is in effect, that would be better.
+    // Then firefox could be fast on pages without CSP.
+    this.isFirefox = navigator.userAgent.indexOf("Firefox") != -1;
```

Comment:
(nitpick) javascript is annoying
```suggestion
    this.isFirefox = navigator.userAgent.indexOf("Firefox") !== -1;
```

Reply:
doh yep typo